### DISCUSSION TITLE
[@types/node-telegram-bot-api]  have some incorrect type

### DISCUSSION THREAD

**Discussion by qiaoshouzi** [DefinitelyTyped/DefinitelyTyped#70041] [Issues with a @types package]
According to the Telegram API doc, `createForumTopic` returns a ForumTopic object.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/5be58a75483be41164b64f7bf57d7d57e1460a93/types/node-telegram-bot-api/index.d.ts#L1919-L1923

https://core.telegram.org/bots/api#createforumtopic
https://core.telegram.org/bots/api#forumtopic

[a comment by an automated account is not shown]

**Comment by qiaoshouzi** [comment]
Just now, I found another incorrect type.

`TelegramBot.ForwardMessageOptions` does not include `message_thread_id`

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/5be58a75483be41164b64f7bf57d7d57e1460a93/types/node-telegram-bot-api/index.d.ts#L170-L173

https://core.telegram.org/bots/api#forwardmessage

**Comment by qiaoshouzi** [comment]
type `Message` does not include `forward_orgin: MessageOrigin`

https://core.telegram.org/bots/api#message
https://core.telegram.org/bots/api#messageorigin

**Comment by XieJiSS** [comment]
Nice catch! Would you mind submitting a PR for this? Also, as a reminder, we'll need to check if the underlying node-telegram-bot-api library supports these attributes as well, even though they are already mentioned in the official bot api docs. This is because the typing needs to describe the library's behavior at the end of the day.

**Reply by qiaoshouzi** [reply]
I open a new PR in #70044

By the way, I found these issues in my project development, so node-telegram-bot-api should be supportive

**Comment by HoseanRC** [comment]
Ok, so this problem is actually more than just a simple function with wrong types, but the type definition file is written for 0.64.0, while the library is at 0.67.0
There are many differences and addition which I'm trying to rewrite based on automatically generated code from the actual official api document of telegram
Im gonna create a whole discussion for this
